.github/workflows/build_wheels.yml

```diff
@@ -15,38 +15,52 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macOS-10.15, ubuntu-20.04, windows-2019]
-
+        os: [ubuntu-latest, macos-latest, windows-latest]
     steps:
       - uses: actions/checkout@v2
 
-      # Used to host cibuildwheel
       - uses: actions/setup-python@v2
 
       - name: Install cibuildwheel
         run: python -m pip install cibuildwheel==1.9.0
 
+      - name: Clone pybind11 repo
+        run: git clone https://github.com/pybind/pybind11.git
+
       - name: Install sundials on macOS
         if: matrix.os == 'macOS-10.15'
         run: |
           brew update
           brew install sundials
 
-      - name: Clone pybind11 repo
-        run: git clone https://github.com/pybind/pybind11.git
-
-      - name: Build wheels on manylinux and macos
-        if: matrix.os != 'windows-2019'
+      - name: Build wheels on Linux and MacOS
         run: python -m cibuildwheel --output-dir wheelhouse
         env:
           CIBW_BEFORE_ALL_LINUX: "bash build_manylinux_wheels/install_sundials.sh 5.8.1 5.7.0"
           CIBW_BEFORE_BUILD_LINUX: "python -m pip install cmake"
 
-      - name: Build wheels on windows
-        if: matrix.os == 'windows-2019'
-        run: |
-          python -m pip install wheel
-          python -m pip wheel --no-deps --wheel-dir wheelhouse .
+      - name: Cache packages installed through vcpkg on windows
+        if: matrix.os == 'windows-latest'
+        uses: actions/cache@v2
+        env:
+          cache-name: vckpg_binary_cache
+        with:
+          path: C:\Users\runneradmin\AppData\Local\vcpkg\archives
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('vcpkg.json') }}
```

Comment:
May (?) also be worth adding the compiler version to the cache key

Reply:
Good idea. Hashing `vcpkg.json` might also not be a great, since it contains the version of PyBaMM. We don't want to clear the cache each time PyBaMM changes version...
I will probably look at this in more details after this is merged.